ESP32: Temperature-measurement-app and lock-app failed to compile.

#### Problem
With respect to the commit 6cbb522b3ac3a6e97400466545429164ab376d60, `temperature-measurement-app` and `lock-app` fail to build successfully.
1. The `temperature-measurement-app` shows below error
```
../main/CHIPDeviceManager.cpp:63:32: note:   crosses initialization of 'chip::RendezvousInformationFlags flags'
     RendezvousInformationFlags flags = RendezvousInformationFlags(CONFIG_RENDEZVOUS_MODE);
                                ^~~~~
```
2. The `lock-app` shows below error.
```
../main/CHIPDeviceManager.cpp:54:13: error: expected ';' before 'RendezvousInformationFlags'
     mCB = cb RendezvousInformationFlags flags = RendezvousInformationFlags(CONFIG_RENDEZVOUS_MODE);
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~
             ;
```

 #### Proposed Solution
Add the needed fix in each example for `CHIPDeviceManager.cpp`.

--- a/examples/lock-app/esp32/main/CHIPDeviceManager.cpp
+++ b/examples/lock-app/esp32/main/CHIPDeviceManager.cpp
@@ -51,7 +51,8 @@ void CHIPDeviceManager::CommonDeviceEventHandler(const ChipDeviceEvent * event, 
 CHIP_ERROR CHIPDeviceManager::Init(CHIPDeviceManagerCallbacks * cb)
 {
     CHIP_ERROR err;
-    mCB = cb RendezvousInformationFlags flags = RendezvousInformationFlags(CONFIG_RENDEZVOUS_MODE);
+    mCB                              = cb;
+    RendezvousInformationFlags flags = RendezvousInformationFlags(CONFIG_RENDEZVOUS_MODE);
 
     // Initialize the CHIP stack.
     err = PlatformMgr().InitChipStack();

--- a/examples/temperature-measurement-app/esp32/main/CHIPDeviceManager.cpp
+++ b/examples/temperature-measurement-app/esp32/main/CHIPDeviceManager.cpp
@@ -54,13 +54,13 @@ void CHIPDeviceManager::CommonDeviceEventHandler(const ChipDeviceEvent * event, 
 CHIP_ERROR CHIPDeviceManager::Init(CHIPDeviceManagerCallbacks * cb)
 {
     CHIP_ERROR err;
-    mCB = cb;
+    mCB                              = cb;
+    RendezvousInformationFlags flags = RendezvousInformationFlags(CONFIG_RENDEZVOUS_MODE);
 
     // Initialize the CHIP stack.
     err = PlatformMgr().InitChipStack();
     SuccessOrExit(err);
 
-    RendezvousInformationFlags flags = RendezvousInformationFlags(CONFIG_RENDEZVOUS_MODE);
     if (flags.Has(RendezvousInformationFlag::kBLE))
     {
         ConnectivityMgr().SetBLEAdvertisingEnabled(true);

